apache/metron
# METRON-2164 Remove the Split-Join Enrichment Topology

The Split-Join Enrichment topology has been deprecated since November 2018. Metron defaults to using the Unified Enrichment topology currently.  This PR completely removes the Split-Join topology from the code base.

Here is the [original discuss thread ](https://lists.apache.org/thread.html/6cfc883de28a5cb41f26d0523522d4b93272ac954e5713c80a35675e@%3Cdev.metron.apache.org%3E)on deprecation.

## Testing

1. Launch the CentOS 6 development environment.

1. Ensure telemetry is being indexed and can be found via the Alerts UI.

1. Go to Ambari > Metron > Configs > Enrichment > Topology and change some of the settings on this panel.  Restart the Enrichment topology and ensure that the settings have taken effect.

1. Again, ensure telemetry is being indexed and indirectly that the Enrichment topology restarted correctly.

## Pull Request Checklist
